docusaurus.config.js

```diff
@@ -1,20 +1,43 @@
 // @ts-check
 // Note: type annotations allow type checking and IDEs autocompletion
 
-const lightCodeTheme = require('prism-react-renderer/themes/github');
-const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+const {themes} = require('prism-react-renderer');
+const lightTheme = themes.github;
+const darkTheme = themes.dracula;
 
 /** @type {import('@docusaurus/types').Config} */
 const config = {
-
+  themes: [
+    [
+      require.resolve("@easyops-cn/docusaurus-search-local"),
+      ({
+        // ... Your options.
+        // `hashed` is recommended as long-term-cache of index file is possible.
+        hashed: true,
+        searchResultContextMaxLength: 100,
+        indexBlog: false,
+        // For Docs using Chinese, The `language` is recommended to set to:
+        // ```
+        language: ["en"],
+        // ```
+      }),
+    ],
+  ],
   plugins: [
+    require.resolve('docusaurus-plugin-image-zoom'),
     [
-      require.resolve('@cmfcmf/docusaurus-search-local'),
+      '@scalar/docusaurus',
       {
-        indexBlog: false,
+        label: 'API Reference',
+        route: '/api',
+        configuration: {
+          spec: {
+            // Put the URL to your OpenAPI document here:
+            url: 'https://raw.githubusercontent.com/bucketeer-io/bucketeer/main/api-description/openapi.yaml'
```

Comment:
@cre8ivejp, to update the OpenAPI URL when using Scalar Plugin, you must update the` docusaurus.config.js` file.